x-pack/plugins/maps/public/classes/sources/setup_sources.ts

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { SOURCE_TYPES } from '../../../common/constants';
+import { registerSource } from './source_registry';
+import { EMSFileSource } from './ems_file_source';
+import { EMSTMSSource } from './ems_tms_source';
+import { ESGeoGridSource } from './es_geo_grid_source';
+import { ESGeoLineSource } from './es_geo_line_source';
+import { ESPewPewSource } from './es_pew_pew_source';
+import { ESSearchSource } from './es_search_source';
+import { GeoJsonFileSource } from './geojson_file_source';
+import { KibanaTilemapSource } from './kibana_tilemap_source';
+import { MVTSingleLayerVectorSource } from './mvt_single_layer_vector_source';
+import { WMSSource } from './wms_source';
+import { XYZTMSSource } from './xyz_tms_source';
+
+const registered = false;
+
+export function setupSources() {
```

Comment:
Nice to see all of these registered in one place! Nit, this `registered` variable doesn't seem to be used at all.

Reply:
thanks. I have updated setup_sources to set `registered = true;` at the end of the method.